### ISSUE TITLE
[ic-page-header] storybook accessibility plug in complaining of two elements with role="banner"

### ISSUE THREAD

**Issue by GCHQ-Developer-530** [mi6/ic-ui-kit#1908]
## Summary of the bug 
On storybook examples of page header with top nav, the built in storybook accessibility plug in is failing due to two elements having a banner role and all roles should be unique. 

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Go to page-header sticky-with-content-and-footer example
2. Click on accessibility plug in in the drawer
4. See the error 

alternatively the error can be replicated on our https://design.sis.gov.uk/top-navigation-page-header-pattern/
If you do a full page scan using the 'axe Dev Tools' browser plugin, it appears as a moderate issue.
If you do an Accessibility Assessment using the IBM Equal Access Accessibility Checker it also appears.

## 📸 Screenshots or code 
<img width="500" alt="Screenshot 2024-05-23 at 14 38 12" src="https://github.com/mi6/ic-ui-kit/assets/111882034/cb9f1599-45c3-4aed-bae4-9335d8d9b0aa">

## 🧐 Expected behaviour 
It should only have one banner.

## Additional info 
Weirdly page header doesn't have a banner role. This doesn't fail accessibility insights fast pass either so it's hard to tell if it's a weird quirk of storybook or not.

**Comment by gd2910** [comment]
Investigate ways around this, consider https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role

Or conditional role on IcPageHeader when in conjunction with IcTopNavigation

**Comment by gd2910** [comment]
Test IcTopnavigation and IcPageHeader layout patterns to recreate issue.

**Comment by GCHQ-Developer-299** [comment]
Error can be replicated on our https://design.sis.gov.uk/top-navigation-page-header-pattern/ 
If you do a full page scan using the 'axe Dev Tools' browser plugin, it appears as a moderate issue.
If you do an Accessibility Assessment using the IBM Equal Access Accessibility Checker it also appears.

**Comment by MI6-255** [comment]
Take banner off Page Header and find a role to replace it
